Cargo.toml

```diff
@@ -24,32 +24,30 @@ log = "0.4"
 thiserror = "1.0"
 
 [dependencies.ring]
-version = "0.16"
+version = "0.17"
 optional = true
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-ring = { version = "0.16", features = ["wasm32_c"], optional = true }
+ring = { version = "0.17", features = ["wasm32_unknown_unknown_js"], optional = true }
```

Comment:
do we have an open source way to varify that this still works afterwards?

Reply:
Unfortunately we can only verify that it still compiles :/ 
But I will double check with our closed source WASM implementation.

Reply:
Maybe we should add an example/test with nodejs for the future.

Reply:
Maybe we should add a minimal wasm example ?

Reply:
Works like a charm btw ;)